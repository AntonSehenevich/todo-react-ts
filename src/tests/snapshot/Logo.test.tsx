import renderer from 'react-test-renderer'
import Logo from '../../components/header/logo'

test('renders correctly', () => {
  const tree = renderer.create(<Logo width={10} height={10} />).toJSON()

  expect(tree).toMatchSnapshot()
})
