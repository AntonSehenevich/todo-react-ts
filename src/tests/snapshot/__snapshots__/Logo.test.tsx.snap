// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  data-testid="logo-icon"
  height={10}
  viewBox="-10 10 50 40"
  width={10}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M 5 45 H 25 V 5"
    fill="transparent"
    stroke="currentColor"
    strokeLinecap="round"
    strokeWidth="2"
    transform="rotate(45 15 25)"
  />
</svg>
`;
